### PR TITLE
chore(RHTAPWATCH-664): Package & Containerize

### PR DESCRIPTION
**Pull request by ifireball**
Adding files to package the tools and deploy into a container image.

Also added a little `odcs_ping` tool to help checking for ODCS connectivity.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
 * yum packages: krb5-devel, gcc, python3.11-devel
+
+## Working with the container image
+
+### To build this repo into a container:
+```
+podman build -t rhtap-tools .
+```
+
+### To run the built container an gain an interactive shell:
```
Comment (yftacherzog) [inline CONTRIBUTING.md:15]:
```suggestion
### To run the built container and gain an interactive shell:
```

**Comment by gbenhaim** [comment]
I agree with @yftacherzog, let's use `appstudio` instead of `rhtap`.

**Comment by ifireball** [comment]
> I agree with @yftacherzog, let's use `appstudio` instead of `rhtap`.

Fixed.

**Review by Omeramsc — APPROVED** [review]
LGTM

**Comment by ifireball** [comment]
@yftacherzog , @gbenhaim , @avi-biton who can merge this?

**Pull request merged**